test/clj/rems/test_text.clj

```diff
@@ -34,7 +34,9 @@
           (is (valid-localization? (localize-state s))))))))
 
 (deftest test-all-event-localizations
-  (let [event-types (keys events/event-schemas)]
+  (let [event-types (-> events/event-schemas
+                        (dissoc :application.event/decided)
```

Comment:
not sure I like how decided is such a special case

Reply:
Done, it's now treated similarly to the other events, except that in the front-end an extra div element is shown, showing the decision.